web/themes/new_weather_theme/templates/block/block--weathergov-local-alerts.html.twig

```diff
@@ -105,20 +105,14 @@
                 <li class="">{{ areaName }}</li>
               {% endfor %}
             </ul>
-          {% endif %}
+            {% endif %}
 
-          <div class="grid-container margin-top-2 padding-x-0 tablet:padding-x-2">
-            <div class="grid-container padding-0 wx-isolation-isolate grid-col-12">
```

Comment:
We'll want to move the `wx-isolation-isolate` class from here and put it on the `wx-alert-map` element